Updated SonarLint

fbernhart opened the pull request:
![sonarlint](https://user-images.githubusercontent.com/70264417/100556969-cfa6c700-329d-11eb-81d3-9ee3b3da9b33.png)


**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

adamrusted reviewed (changes requested):
Repo that is currently source has been archived. The SVG logo is embedded in the homepage of [SonarSource](https://www.sonarsource.com/), could you update source to this, and compare PR logo against new source? 🙏🏻

fbernhart commented:
@adamrusted Done 👍🏼  I've changed the source, rebuilt from it and changed the color as well (used the color from the [SVG](https://www.sonarsource.com/index/products/sonarlint-logo-black.svg)).

![sonarlint](https://user-images.githubusercontent.com/70264417/103233492-949bc000-493d-11eb-9e5a-951801481a30.png)

adamrusted reviewed (approved):
Perfect! 👌🏻
